doc/src/index.rst

```diff
@@ -21,6 +21,7 @@ Contents:
    user_guide
    developer_guide
    examples
+   performance_tuning
```

Comment:
Maybe worth putting the performance tuning before the examples

Reply:
Moved